library.properties

```
@@ -1,9 +1,10 @@
 name=ConfigTool
-version=1.0
-author=Tvde1
-maintainer=Tvde1
+version=1.1
+author=Tvde1,gerdlanger
+maintainer=gerdlanger
```

```suggestion
maintainer=Tvde1
```
I very much appreciate the effort you have put in so it's very fair you are listed as author but I think that I should still be listed as the maintainer. I promise I'm not gonna abandon this library anytime soon :)